src/main/scala/reactive/Main.scala

```diff
@@ -0,0 +1,96 @@
+package reactive
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.marshalling.ToResponseMarshallable.apply
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directive.{addByNameNullaryApply, addDirectiveApply}
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.PathMatcher.segmentStringToPathMatcher
+import akka.http.scaladsl.server.PathMatchers.Segment
+import akka.http.scaladsl.server.Route
+import akka.http.scaladsl.server.RouteResult.route2HandlerFlow
+import akka.pattern.ask
+import akka.stream.ActorFlowMaterializer
+import akka.util.Timeout
+import reactive.tweets.domain.{Tweet, User}
+import reactive.tweets.incoming.TweetActor.{GetLastTen, LastTenResponse}
+import reactive.tweets.incoming.TweetActorManager
+import reactive.tweets.outgoing.TweetFlow
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.DurationInt
+import scala.language.postfixOps
+
+object Main extends App with TweetFlow {
+  implicit val system = ActorSystem("webapi")
+  implicit val executor = system.dispatcher
+  implicit val timeout = Timeout(1000 millis)
```

Comment:
Postfix operators have been deprecated for a reason. Given that this thing basically serves to show people how things should be done, I think we should remove all `scala.language.postfixOps` references. It's not really so bad to write `1000.millis` instead of `1000 millis`, is it?